core/server/apps/private-blogging/lib/views/private.hbs

```diff
@@ -15,7 +15,7 @@
         <link rel="shortcut icon" href="{{asset "favicon.ico"}}">
         <meta http-equiv="cleartype" content="on">
 
-        <link rel="stylesheet" href="{{asset "ghost.css" ghost="true" minifyInProduction="true"}}" />
```

Comment:
I don't think the asset helper can be removed here? 

Has this PR been tested on a subdirectory setup?